src/components/BusinessOnboarding/CompanyCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {
+    CardWrapper,
+    Title,
+    Label,
+    Input,
+    Button,
+
+} from './BusinessOnboarding.styles';
+
+const CompanyCard = ({ title, input, button }) => {
```

Comment:
just accept a props object here and decompose on the next line instead of doing it all inline